com.unity.render-pipelines.high-definition/Documentation~/Decal-Shader.md

```diff
@@ -46,10 +46,12 @@ These properties allow you to set the inputs that affect the behavior of the dec
 
 These properties allow you to change the rendering behavior of the decal.
 
-| **Property**              | **Description**                                              |
-| ------------------------- | ------------------------------------------------------------ |
-| **Draw Order**            | Controls the order in which HDRP draws decals in the Scene. HDRP draws decals with lower values first, so it draws decals with a higher draw order value on top of those with lower values. This feature works for decals projected on opaque and transparent surfaces.<br />**Note**: This property only applies to decals the [Decal Projector](Decal-Projector.md) creates and has no effect on Mesh decals. Additionally, if you have multiple Decal Materials with the same **Draw Order**, the order HDRP renders them in depends on the order you create the Materials. HDRP renders Decal Materials you create first before those you create later with the same **Draw Order**. |
-| **Mesh Decal Depth Bias** | A depth bias that HDRP applies to the decal’s Mesh to stop it from overlapping with other Meshes. A negative value draws the decal in front of any overlapping Mesh, while a positive value offsets the decal and draw it behind. This property only affects decal Materials directly attached to GameObjects with a Mesh Renderer, so Decal Projectors do not use this property. |
+| **Property**               | **Description**                                              |
+| -------------------------- | ------------------------------------------------------------ |
+| **Draw Order**             | Controls the order in which HDRP draws decals in the Scene. HDRP draws decals with lower values first, so it draws decals with a higher draw order value on top of those with lower values. This feature works for decals projected on opaque and transparent surfaces.<br />**Note**: This property only applies to decals the [Decal Projector](Decal-Projector.md) creates and has no effect on Mesh decals. Additionally, if you have multiple Decal Materials with the same **Draw Order**, the order HDRP renders them in depends on the order you create the Materials. HDRP renders Decal Materials you create first before those you create later with the same **Draw Order**. |
+| **Mesh Decal Bias Type**   | Determines the type of bias that HDRP applies to the decal’s Mesh to stop it from overlapping with other Meshes. Depth Bias applies a bias to the final depth value, while View bias applies a world space bias (in meters) alongside the view vector. |
```

Comment:
Determines the type of bias that HDRP applies to the decal’s Mesh to stop it from overlapping with other Meshes. The options are:<br/>&#8226; **Depth Bias**: Applies a bias to the final depth value, <br/>&#8226; **View Bias**: Applies a world-space bias (in meters) alongside the view vector.